src/Monetization.php

```diff
@@ -237,6 +239,48 @@ public function apiProductAssignmentAccess(EntityInterface $entity, AccountInter
     }
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function apiProductTeamAssignmentAccess(ApiProductInterface $api_product, TeamInterface $team, AccountInterface $account): AccessResultInterface {
```

Comment:
These changes needs to be added in m10n teams module.

Reply:
Function called from m10n teams module